lib/pdk/validators/base_validator.rb

```diff
@@ -4,6 +4,8 @@
 module PDK
   module Validate
     class BaseValidator
+      INVOKE_STYLE = :once
```

Comment:
I'm unsure if the intention of _:once_ is immediately understandable - might _:combined_ or _:all_targets_ be more immediately understandable?
Or leave it as _:once_ but add some commenting here on the possible values of INVOKE_STYLE